tests/schema_update_test.go

```diff
@@ -26,6 +32,17 @@ func TestSchemaUpdate(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	license, err := fs.ReadFile(fakeFS, "fake/license")
```

Comment:
Is this required for each test, or is this something we'd like to test? For latter, we can introduce a new test.

Reply:
It's required as we need to create a new instance. We also should add new tests for different subscription plan

Reply:
SG